lib/charms/tempo_k8s/v1/charm_tracing.py

```diff
@@ -244,11 +267,9 @@ def wrap_init(self: CharmBase, framework: Framework, *args, **kwargs):
         server_cert: Optional[str] = (
             _get_server_cert(server_cert_getter, self, charm) if server_cert_getter else None
         )
-        credentials = ChannelCredentials(server_cert) if server_cert else None
-        insecure = None if credentials else True
 
         exporter = OTLPSpanExporter(
-            endpoint=tracing_endpoint, credentials=credentials, insecure=insecure, timeout=2
+            endpoint=tracing_endpoint, certificate_file=server_cert, timeout=2
```

Comment:
```suggestion
            endpoint=tracing_endpoint, certificate_file=str(Path(server_cert).abspath()), timeout=2
```

Reply:
Done with a tiny difference as `Path` has `.absolute()` function that does the same.